amqtt/contrib/shadows/messages.py

```diff
@@ -0,0 +1,107 @@
+from collections.abc import MutableMapping
+from dataclasses import dataclass, fields, is_dataclass
+import json
+from typing import Any
+
+from amqtt.contrib.shadows.states import MetaTimestamp, ShadowOperation, State, StateDocument
+
+
+def asdict_no_none(obj: Any) -> Any:
```

Comment:
How about 
```suggestion
def as_dict_no_none(obj: Any) -> Any:
```
to keep consistency of splitting words with underbars?

Reply:
Ah that comes from `dataclasses.asdict`. OK